src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly

```diff
@@ -92,11 +92,29 @@
     <span>
       <a href="https://github.com/jenkinsci/jenkins/tree/master/war/src/main/resources/images/symbols"
          target="_blank" class="jdl-paragraph jdl-link">
-        ${%View the complete list of Jenkins Symbols on GitHub}
+        ${%View the complete list of symbols provided by Jenkins on GitHub}
         <l:icon src="symbol-external plugin-design-library" />
       </a>
     </span>
 
+    <h2 class="jdl-heading">${%ioniconsApi}</h2>
+    <p class="jdl-paragraph">${%customSymbols.ionicons-api}</p>
+
+    <span>
+      <a href="https://plugins.jenkins.io/ionicons-api/#dependencies" target="_blank" class="jdl-paragraph jdl-link">
```

Comment:
Page is not visible yet and we should hold off publishing this till present

Reply:
Yeah 👍🏻 

How often does the update center job run? The initial release was ~16h ago, but https://updates.jenkins-ci.org/current/update-center.json doesn't offer it yet.

Reply:
should be done, I would raise a helpdesk ticket